source/_integrations/esphome.markdown

```diff
@@ -42,6 +42,9 @@ works_with:
 
 This integration allows [ESPHome](https://esphome.io) devices to connect directly to Home Assistant with the [native ESPHome API](https://esphome.io/components/api.html).
 
+
+`Note: This system uses MDNS for ease of use on Home networks. Multiple subnets and vlans will require the use of additional options for devices to show online. Please see the forums for needed additions to configurations to allow devices to be detected.`
```

Comment:
This is an advanced networks setup, which we generally do not support. If you manage/set up such networks, we will assume you are capable of working with those and knows what that entails.

Additionally, this note is not about Home Assistant, but about the ESPHome Dashboard itself; and thus, IMHO, this is not the right place to add this.

../Frenck

Reply:
thats quite the unfrendly attitude to bring towards people just because we all cant just download 20 years of development knowledge of  the esphome project from day one like you have.  i know of DBA's who are quite increadible at building a massive enterprise sql database system and have troubles with turning on the pc they work on or following prompts of software installs.  

This is the most appropriate location as it pertains to Home Assistant users who are looking to add functionality and would most benifit from being aware of the addon's limitations/design choices as it pertains to them and their intrest of adding it to their setup.

Reply:
> thats quite the unfrendly attitude

Sorry, it isn't meant like that, nor did I put it like that.

> This is the most appropriate location as it pertains to Home Assistant users who are looking to add functionality and would most benifit from being aware of the addon's limitations/design choices as it pertains to them and their intrest of adding it to their setup.

Sorry, I'm in disagreement, as you are documenting a behavior in an advanced network setup about the ESPHome Dashboard. Not Home Assistant.

> and would most benifit from being aware of the addon's limitations/design choices as it pertains to them and their intrest of adding it to their setup.

It is not an add-on specific limitation. It is a limitation of your network setup, which will occur in many other things, including non-add-on usage of ESPHome.

../Frenck